styles/command-palette.less

```diff
@@ -0,0 +1,7 @@
+@import "ui-variables";
+
+// Highlight matched text
+.select-list .list-group .character-match {
```

Comment:
This should probably be under a `.command-palette {` rule so this rule doesn't affect other select lists.